tests/test_failure_propagator.py

```diff
@@ -1,15 +1,83 @@
 import pytest
 
-from ocs_ci.framework.pytest_customization.marks import tier1
+from ocs_ci.framework import config
+from ocs_ci.framework.pytest_customization.marks import (
+    acceptance,
+    tier1,
+    tier2,
+    tier3,
+    tier4a,
+    tier4b,
+    tier4c,
+    pre_upgrade,
+    post_upgrade,
+    pre_ocs_upgrade,
+    pre_ocp_upgrade,
+    post_ocp_upgrade,
+    post_ocs_upgrade,
+    workloads,
+    performance,
+    scale,
+)
 
 
 @tier1
-@pytest.mark.last
+@acceptance
+@tier2
+@tier3
+@tier4a
+@tier4b
+@tier4c
+@pre_upgrade
+@post_upgrade
+@pre_ocs_upgrade
+@pre_ocp_upgrade
+@post_ocp_upgrade
+@post_ocs_upgrade
+@workloads
+@performance
+@scale
 class TestFailurePropagator:
     """
-    Test class for failure propagator test case. The test intention is to run last and propagate
-    teardown failures caught during the test execution, so regular test cases won't false negatively fail
+    Test class for failure propagator test case
     """
 
+    @pytest.mark.second_to_last
+    def test_report_skip_triggering_test(self, request):
+        """
+        This test runs second to last and examines the skipped test cases of the execution.
+        In case of high rate of skipped tests due to Ceph health not OK, which indicates something went wrong
+        with the cluster during the execution, it will fail and report the potential test case that caused
+        this problematic state
+        """
+        number_of_eligible_tests = config.RUN.get("number_of_tests") - 2
+
+        skipped_on_ceph_health_percent = (
+            config.RUN.get("skipped_tests_ceph_health") / number_of_eligible_tests
+        )
+        message = (
+            f"This run had {skipped_on_ceph_health_percent * 100}% of the "
+            f"tests skipped due to Ceph health not OK."
+        )
+        if 0.5 < skipped_on_ceph_health_percent < 1:
```

Comment:
What is the reason for the second part of the condition `skipped_on_ceph_health_percent < 1`?
I meant, that it could theoretically be `1` when all the tests will be skipped and we probably don't want to miss that situation.

Also I'm thinking if 50% skipped tests is ok?

Reply:
you are right. Let me remove the `< 1` part and reduce the threshold to 25%